src/lints/function_parameter_count_changed.ron

```diff
@@ -51,4 +55,7 @@ SemverQuery(
     },
     error_message: "A publicly-visible function now takes a different number of parameters.",
     per_result_error_template: Some("{{join \"::\" path}} now takes {{current_parameter_count}} parameters instead of {{old_parameter_count}}, in {{span_filename}}:{{span_begin_line}}"),
+    witness: (
+    hint_template: r#"let witness = |{{join " , " old_parameter_names}}| {{join "::" path}}({{join " , " old_parameter_names}});"#,
```

Comment:
Let's leave space only after the comma, not before. This is also what `cargo fmt` does so we should adhere to it as well.